Mods/vcmi/config/vcmi/english.json

```diff
@@ -59,6 +59,13 @@
 
 	"vcmi.spellBook.search" : "search...",
 
+	"vcmi.spellResearch.canNotAfford" : "You can't afford to replace {%SPELL1} with {%SPELL2}. But you can still discard this spell and continue spell research.",
```

Comment:
I though that discarding / continuing spell research also costs resources?

Reply:
Currently not.